### ISSUE TITLE
Custom executor - docs - example in language different than Go

### ISSUE THREAD

**Issue by exu** [kubeshop/testkube#925]
Languages ?
- [ ] Rust
- [x] nodejs 
- [ ] python
- [ ] java 
- [ ] php?
- [ ] Kotlin

**Comment by exu** [comment]
blocked by #979

**Comment by exu** [comment]
would be nice to have filesystem abstraction in volume first

**Comment by Itayturni** [comment]
do you have a python example?

**Comment by TheBrunoLopes** [comment]
Hello @Itayturni we don't have a python example right now.
But we can help you create the executor you need. Have you taken a look at [container executors](https://kubeshop.github.io/testkube/test-types/container-executor) ?
I would be more than happy to help you with any obstacle, feel free to book a quick call with me [here](https://calendly.com/bruno-at-kubeshop/30min-1)
